src/lib/decorators.ts

```diff
@@ -265,3 +265,28 @@ export function eventOptions(options: AddEventListenerOptions) {
              // tslint:disable-next-line:no-any decorator
              any;
 }
+
+/**
+ * A property decorator that converts a class property into a getter that
+ * returns the `assignedNodes` of the given named `slot`.
```

Comment:
I think we should document the type that the property should be annotated with, ie `NodeListOf<HTMLElement>`

Reply:
Done